### DISCUSSION TITLE
prisma models in separate files

### DISCUSSION THREAD

**Discussion by slytherin-afk** [prisma/prisma#16992] [Q&A]
How can I write my models in separate files?

**Comment by nurul3101** [comment]
Hello @mraza30 👋 
You would need to use community-built tools like [Aurora](https://github.com/sabinadams/aurora), or recently released [prisma-import](https://github.com/ajmnz/prisma-import) which resolves this issue.

We have a Feature Request for splitting schema files
- https://github.com/prisma/prisma/issues/2377

I would recommend you to give a 👍 to the feature request so our product team can prioritise it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
